Vagrantfile

```diff
@@ -156,6 +157,7 @@ end
 
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+    config.vm.box_check_update = false
```

Comment:
That sounds good. @DivyaVavili let's do that instead of combining with this commit.